lib/plug/static.ex

```diff
@@ -7,12 +7,12 @@ defmodule Plug.Static do
     * `:at` - the request path to reach for static assets.
       It must be a string.
 
-    * `:from` - the filesystem path to read static assets from.
-      It must be a string, containing a file system path, or an
-      atom representing the application name where assets will
-      be served from the priv/static, or a tuple containing the
-      application name and the directory to serve assets besides
-      priv/static.
+    * `:from` - the file system path to read static assets from.
```

Comment:
Oh sorry, I came out wrong, definitely no one will be confused! :D I only meant to say that since we haven't a standard of saying "filesystem" or "file system" in the docs (for Plug, or Elixir), this change felt a bit arbitrary. Sorry for the cryptic way of saying this 😄